### PR TITLE
UofT-DSI | deep learning - Assignment 1

### PR DESCRIPTION
**Pull request by ShayanHodai**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
First, I built a base CNN model, then tried to enhance it by adding a Dropout layer to improve performance.

## What did you learn from the changes you have made?
I learned that adding a dropout layer can help prevent overfitting but might lower the accuracy if not tuned correctly.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I was also thinking about increasing epochs or reducing the dropout layer to 0.2

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
After adding the dropout layer, the model's accuracy didn't improve as expected. I started thinking about why this happened and considered different factors, like the dropout rate and the possibility that the model might need further tuning or adjustments.

## How were these changes tested?
I trained both models (original and enhanced) and compared their performance on the test set.

## A reference to a related issue in your repository (if applicable)
N/A

## Checklist
- [ x] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by ananyajha2000** [comment]
A1 Complete
